clap_builder/src/output/help_template.rs

```diff
@@ -1142,6 +1169,10 @@ fn should_show_subcommand(subcommand: &Command) -> bool {
     !subcommand.is_hide_set()
 }
 
+fn should_show_help_heading(subcommand: &Command) -> bool {
+    subcommand.is_help_heading_set()
+}
+
```

Comment:
This is a pretty thin wrapper, not sure it is providing enough value

Reply:
Ok, I will remove.